gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/impl/sql/execute/TablePrivilegeInfo.java

```diff
@@ -222,20 +227,48 @@ public void executeGrantRevoke( Activation activation,
 									List grantees)
 		throws StandardException
 	{
+		doExecuteGrantRevoke(activation, grant, grantees, columnBitSets, actionAllowed, true, td);
+		GemFireStore ms = Misc.getMemStore();
+		// TODO ashetkar Get a better way to check if security is enabled.
+		if (ms.isSnappyStore() && "LDAP".equalsIgnoreCase(ms.getBootProperty(Attribute.AUTH_PROVIDER))
```

Comment:
@kneeraj Didn't got your comment. 

However I was thinking that above method throws an exception. Is is fine or a similar no throw method be implemented in Misc, and used here checking for null.